docs/changelog.md

```diff
@@ -120,6 +120,15 @@ what we publish.
   determining a default SDK to use. The user can select the default SDK to use
   with the `Mojo: Select the default MAX SDK` command.
 
+- Mojo-Python slice interoperability, enabling something like below:
+
+  ```mojo
+  a = PythonObject([1, 2, ,3, 4])
+  a_reversed = a[::-1]
+  ```
+
```

Comment:
nit (changelog file is very long already):
```suggestion
  ```mojo
  print(PythonObject([1, 2, ,3, 4])[::-1]) # [4, 3, 2, 1]
  ```
```